### ISSUE TITLE
Fix feature service issue where geometries can be altered

### ISSUE THREAD

**Issue by CoreyKrewson-NOAA** [NOAA-OWP/hydrovis#489]
I noticed that the services with the feature access enabled are allowing geometry updates which we don't want. We will need to update the "update_data_stores_and_sd_files.py" to fix that issues

**Comment by shawncrawley** [comment]
I'm unsure what change would be required. It looks to me like the behavior in the code is already as you describe it needing to be:
https://github.com/NOAA-OWP/hydrovis/blob/0a0c3aa2e94789837fc61c9e98535f3e77005701/Source/Visualizations/aws_loosa/deploy/update_data_stores_and_sd_files.py#L323-L324

Am I missing something?

**Comment by CoreyKrewson-NOAA** [comment]
yeah that code is in there but it doesn't seem to be working. I went and updated them all manually in production

**Comment by shawncrawley** [comment]
Ah that makes more sense - thanks for the clarification. I'll look into it.
